src/app/components/signin/signin.component.ts

```diff
@@ -64,18 +56,15 @@ export class SigninComponent implements OnInit {
             message: 'Logged in successfully! Redirecting...'
           };
 
-          // this.api.getToken(); TODO - Remove
-          // this.authService.setLoginStatus(true); TODO - Remove
-
-          this.router.navigateByUrl('/channels');
+          setTimeout(() => {
+            this.router.navigateByUrl('/channels');
+          }, 1000);
 
```

Comment:
Same here. It is either too long, or a potential problem, if we indeed need to wait for something (and it eventually takes more than 1000ms to complete).